phases/ephemeral/witx/typenames.witx

```diff
@@ -276,8 +276,26 @@
     ;;; If `rights::fd_read` is set, includes the right to invoke `poll_oneoff` to subscribe to `eventtype::fd_read`.
     ;;; If `rights::fd_write` is set, includes the right to invoke `poll_oneoff` to subscribe to `eventtype::fd_write`.
     $poll_fd_readwrite
-    ;;; The right to invoke `sock_shutdown`.
-    $sock_shutdown
```

Comment:
Can you comment on why `sock_shutdown` is being removed? `shutdown` is in the traditional Berkeley sockets API.